jsonschema-core.xml

```diff
@@ -2095,6 +2101,442 @@
             </section>
         </section>
 
+        <section title="Output Formatting" anchor="output">
+            <t>
+                JSON Schema is defined to be platform-independent.  As such, to increase compatibility
+                across platforms, implementations SHOULD conform to a standard validation output
+                format.  This section describes the minimum requirements that consumers will need to
+                properly interpret validation results.
+            </t>
+
+            <section title="Format">
+                <t>
+                    JSON Schema output is defined using JSON.  Implementations MAY deviate from this
+                    as supported by their specific languages and platforms, however it is RECOMMENDED
+                    that the output be convertible to the JSON format defined herein via serialization
+                    or other means.
+                </t>
+            </section>
+
+            <section title="Output Levels">
+                <t>
+                    This specification defines four levels of output.  See the *Output Structure*
+                    section for the requirements of each level.
+                    <list>
+                        <t>
+                            Flag - A boolean which simply indicates whether the schema was
+                            validated.
+                        </t>
+                        <t>
+                            Basic - Provides validation information in a flat list structure.
+                        </t>
+                        <t>
+                            Detailed - Provides validation information in a condensed hierarchical
+                            structure based on the structure of the schema.
+                        </t>
+                        <t>
+                            Verbose - Provides validation information in an uncondensed hierarchical
+                            structure that matches the exact structure of the schema.
+                        </t>
+                    </list>
+                    Implementations SHOULD specify which levels of validation they support.
```

Comment:
Do you need to implement all levels up to and including the most complex/verbose one?  Or could an implementation support *only* the Verbose option?  We should be clear about that in this section.

Reply:
I don't think that an "up-to" requirement is necessary.  Implementations should be free to support any of the formats.  Maybe "levels" is wrong?

Reply:
[As I originally described it](https://github.com/json-schema-org/json-schema-spec/issues/396#issuecomment-406128790) an updated version would say that an implementation:
MUST provide flag
SHOULD provide basic
RECOMMENDED to provide detailed and
OPTIONALLY provide verbose

Reply:
If the verbose format is likely to be easier to implement I would prefer to allow any subset of formats be implemented.  @Anthropic do you feel strongly about this?